MeshLib/MeshGenerators/LayeredMeshGenerator.cpp

```diff
@@ -48,7 +48,7 @@ bool LayeredMeshGenerator::createLayers(MeshLib::Mesh const& mesh,
     std::vector<GeoLib::Raster const*> rasters;
     rasters.reserve(raster_paths.size());
     for (auto path = raster_paths.begin(); path != raster_paths.end(); ++path)
-        rasters.push_back(FileIO::AsciiRasterInterface::getRasterFromASCFile(*path));
+        rasters.push_back(FileIO::AsciiRasterInterface::readRaster(*path));
```

Comment:
I recommend to split the changes by topic. This one is not adding a command line tool contrary to the commit message.